addons/crm/views/crm_lead_views.xml

```diff
@@ -633,10 +633,10 @@
                     <div class="o_kanban_record_subtitle">
                         <t t-if="record.prorated_revenue.raw_value">
                             <field name="prorated_revenue" widget="monetary" options="{'currency_field': 'company_currency'}"/>
-                            <span t-if="record.recurring_revenue and record.recurring_revenue.raw_value"> + </span>
+                            <span t-if="record.recurring_revenue_prorated and record.recurring_revenue_prorated.raw_value"> + </span>
```

Comment:
This line diff may not be necessary. There is no need to compute the `_prorated` version if the base `recurring_revenue` is 0, and that one is stored. We already know probability is not `0` (see line 634).
Same applies for line 638.